Makefile

```diff
@@ -0,0 +1,41 @@
+DOCKER_TRUSS_HOME := /go/src/github.com/TuneLab/go-truss
+DOCKER_BUILD_IMAGE_NAME := tunelab/truss-build
+DOCKER_RUN_IMAGE_NAME := tunelab/truss
+NOW := $(shell date -u "+%Y-%m-%dT%H_%M_%S")
+
+# Default target builds Truss in using the truss-build container, then
+# packages the output into a new truss image, and removes the build output
+# from the working directory. Note that if make targets are evaluated in
+# parallel, this target will not work since it depends on the order of the
+# dependent targets as specified.
+default: docker-build build-docker-run
```

Comment:
I feel that by default this should build truss itself as this Makefile lives in the root of the repository.

```
default: 
    $(MAKE) -C truss
```